source/client/service_impl.h

```diff
@@ -17,7 +17,6 @@
 #include "external/envoy/source/common/common/thread.h"
 #include "external/envoy/source/common/event/real_time_system.h"
 #include "external/envoy/source/exe/process_wide.h"
-
```

Comment:
Is this PR updated to the latest on master? I’m surprised the formatting check doesn’t object to this?
Regardless this change doesn’t seem nessecary?

Reply:
Agreed. I will leave it unchanged.

Reply:
PR is updated to the latest on master.